### PR TITLE
misc: add env var to control log level

### PR DESCRIPTION
**Pull request by rpurdel**

### CODE REVIEW

File: skynet/logs.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import logging
 import sys
 from logging import Filter, LogRecord
-
 from uvicorn.logging import DefaultFormatter
+from env import log_level
```
Comment (saghul) [inline skynet/logs.py:5]:
```suggestion
from .env import log_level
```

File: skynet/env.py [hunk 2]
```diff
@@ -3,7 +3,8 @@
 
 is_mac = sys.platform == 'darwin'
 
-# modules
+# general
+log_level = os.environ.get('LOG_LEVEL', 'INFO').strip().upper()
```
Comment (saghul) [inline skynet/env.py:7]: Don't we want to default to debug?

File: skynet/logs.py (hunk 1 above)
Comment (rpurdel) [reply skynet/logs.py]: Used abs import to avoid any confusion

File: skynet/env.py (hunk 2 above)
Comment (rpurdel) [reply skynet/env.py]: defaulted to debug

**Review by saghul — APPROVED** [review]

**Pull request merged**